### ISSUE TITLE
Self-supervised learning through the eyes of a child

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#52]
# TL;DR
This study is an attempt to analyze the development of baby's intelligence using DL. The accuracy of each category is observed by solving classification problems using the representations obtained from the self-supervised learning (MOCO) from the videos of the camera attached to the head to see the accuracy of each category, and what the babies are learning is also observed by visualization with Attention Map.
![image](https://user-images.githubusercontent.com/24524018/90889307-e57d3400-e3f2-11ea-8ae9-c18d299514af.png)
![image](https://user-images.githubusercontent.com/24524018/90889315-e910bb00-e3f2-11ea-9b25-d08a99d64b75.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2007.16189

# Submission Dates(yyyy/mm/dd)
2020/07/31

# Authors and institutions

 A. Emin Orhan, Vaibhav V. Gupta, Brenden M. Lake
- New York University

# Methods

# Results

# Comments
